media-libs/jasper/jasper-2.0.33.recipe

```diff
@@ -0,0 +1,115 @@
+SUMMARY="Implementation of the codec specified in the JPEG-2000 Part-1 standard"
+DESCRIPTION="The JasPer Project is an open-source initiative to provide a \
+free software-based reference implementation of the codec specified in the \
+JPEG-2000 Part-1 standard. More details about this software can be found in \
+the JasPer Software Reference Manual."
+HOMEPAGE="http://www.ece.uvic.ca/~frodo/jasper/"
+COPYRIGHT="2001-2021 Michael D. Adams"
+LICENSE="JasPer v2"
+REVISION="1"
+SOURCE_URI="https://github.com/mdadams/jasper/archive/version-$portVersion.tar.gz"
+CHECKSUM_SHA256="38b8f74565ee9e7fec44657e69adb5c9b2a966ca5947ced5717cde18a7d2eca6"
+SOURCE_DIR="jasper-version-$portVersion"
+
+ARCHITECTURES="all !x86_gcc2"
+SECONDARY_ARCHITECTURES="x86"
+
+libVersion="4.0.0"
+libVersionCompat="$libVersion compat >= ${libVersion%%.*}"
+
+PROVIDES="
+	jasper$secondaryArchSuffix = $portVersion
+	lib:libjasper$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libjpeg$secondaryArchSuffix
+	"
+
+PROVIDES_devel="
+	jasper${secondaryArchSuffix}_devel = $portVersion
+	devel:libjasper$secondaryArchSuffix = $libVersionCompat
+	"
+REQUIRES_devel="
+	jasper$secondaryArchSuffix == $portVersion base
+	"
+
+if [ -z "$secondaryArchSuffix" ]; then
+	SUMMARY_tools="The jasper tools"
+	PROVIDES_tools="
+		jasper${secondaryArchSuffix}_tools = $portVersion
+		cmd:imgcmp$secondaryArchSuffix = $portVersion
+		cmd:imginfo$secondaryArchSuffix = $portVersion
+		cmd:jasper$secondaryArchSuffix = $portVersion
+		cmd:jiv$secondaryArchSuffix = $portVersion
+		"
+	REQUIRES_tools="
+		jasper$secondaryArchSuffix == $portVersion base
+		haiku$secondaryArchSuffix
+		lib:libGL$secondaryArchSuffix
+		lib:libglu$secondaryArchSuffix
+		lib:libjpeg$secondaryArchSuffix
+		"
+fi
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libGL$secondaryArchSuffix
+	devel:libglu$secondaryArchSuffix
+	devel:libjpeg$secondaryArchSuffix
+	"
+BUILD_PREREQUIRES="
+	cmd:cmake
+	cmd:gcc$secondaryArchSuffix
+	cmd:imginfo$secondaryArchSuffix
```

Comment:
This command probably isn't available on 32bit secondary architecture, needs to be checked!